verge3.html

```diff
@@ -0,0 +1,437 @@
+                    <li>Tech</li>
+                    <li>Science</li>
+                    <li>Culture</li>
+                    <li>Cars</li>
+                    <li>Design</li>
+                    <li>Business</li>
+                    <li>US & World</li>
+                </ul>
+                <ul class="right_items">
+                    <li id="hidden_main_li_2"><em>THE BASICS</em></li>
+                    <li>Site Status</li>
+                    <li>About</li>
+                    <li>Contact</li>
+                    <li>Tip Us</li>
+                    <li>Forums</li>
+                    <li>Jobs</li>
+                </ul>
+                <ul class="right_items_two">
+                    <li id="hidden_main_li_3"><em>THE ELSEWHERE</em></li>
+                    <li>Facebook</li>
+                    <li>Twitter</li>
+                    <li>Tumblr</li>
+                    <li>YouTube</li>
+                    <li>Instagram</li>
+                    <li>Subscribe</li>
+                </ul>
+            </div>
+        </section>
+        <ul class="social_networking_sites_plugins">
+            <li><img src="http://hariiyer.com/wp-content/uploads/2014/04/Facebook-Vector-Icon.png"></li>
+            <li><img src="http://hariiyer.com/wp-content/uploads/2014/04/Facebook-Vector-Icon.png"> </li>
+            <li><img src="http://hariiyer.com/wp-content/uploads/2014/04/Facebook-Vector-Icon.png"></li>
+            <li><img src="http://hariiyer.com/wp-content/uploads/2014/04/Facebook-Vector-Icon.png"> </li>
+        </ul>
+        <section class="wrapper_footer_status_corner_wrapper_footer_caption_copy_rights">
+            <div class="footer_status_corner">
+                <a href="#">Ethics Statement</a>
+                <a href="#">Community Guidelines</a>
+                <br>
+                <a href="#">Terms of Use</a>
+                <a href="#">Privacy Policy</a>
+                <br>
+                <span>  
+                                        All Systems Operational
+                                        </span>
+                <a href="#">Check out our status page for more details.</a>
+            </div>
+            <section class="wrapper_footer_caption_copy_rights">
+                <span class="footer_caption">
+                    <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/3/36/Vox_Media_logo.svg/2000px-Vox_Media_logo.svg.png">
+                </span>
+                <div class="copy_rights">
+                    <p>
+                        © 2016 <span>Vox Media</span> , Inc. All rights reserved.
+                </p>
+                </div>
+            </section>
+        </section>
+    </footer>
+    <script>
+    $('.menu_icon').click(function() {
+        $('.animated_vertical_navbar ').removeClass('transform_left');
+        $('.animated_vertical_navbar_shadow ').removeClass('transform_left_shadow');
+
+        $('.animated_vertical_navbar ').addClass('transform_right');
```

Comment:
use a common variable for $('.animated_variable-nav-bar'). use variables for all such occurences